base/applications/cmdutils/more/lang/ro-RO.rc

```diff
@@ -1,60 +1,64 @@
 /*
- * FILE:       base/applications/cmdutils/more/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Fulea Ștefan (PM on ReactOS Forum at fulea.stefan)
- * CHANGE LOG: 2011-08-20  initial translation
- *             2011-10-17  diacritics change, other minor changes
+ * PROJECT:     ReactOS More Command
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Romanian resource file
+ * TRANSLATORS: Copyright 2011-2019 Ștefan Fulea <stefan.fulea@mail.com>
+ *              Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 STRINGTABLE
 BEGIN
-    IDS_USAGE "Afișează date pe ecran, pagină-cu-pagină.\n\
+    IDS_USAGE "Afișează date pe ecran, pagină cu pagină.\n\
 \n\
-MORE [/E [options]] < [drive:][path]filename\n\
-command | MORE [/E [options]]\n\
+MORE [/E [opțiuni]] < [disc:][cale]numele fișierului\n\
+command | MORE [/E [opțiuni]\n\
 \n\
-    [drive:][path]filename  A file whose contents are to be displayed.\n\
-    command                 A command whose output is to be displayed.\n\
+    [disc:][cale]numele fișierului  Un fișier al cărui conținut urmează să fie afișat.\n\
+    comandă                         O comandă a cărei ieșire urmează să fie afișată.\n\
 \n\
-When Extended Features are enabled, the following form is supported:\n\
-MORE [/E] [options] [files]\n\
+Atunci când opțiunea Caracteristici extinse e activată, următoarea formă este
+acceptată:\n\
+MORE [/E] [opțiuni] [fișiere]\n\
 \n\
-Options:\n\
-    /E       Enable Extended Features.\n\
-    The default state of the Extended Features is controlled by the default\n\
-    registry settings for CMD.EXE. For more details on how to change these\n\
-    settings, see the section ""Command Extensions"" in CMD.EXE help.\n\
+Opțiuni:\n\
+    /E       Activează opțiunea Caracteristici extinse.\n\
+    Stadiul implicit al Caracteristicilor extinse e controlat de\n\
+    configurațiile implicite de registru pentru programul CMD.EXE. Pentru mai\n\
+    multe detalii legate de cum pot fi schimbate aceste configurații, vedeți\n\
+    secțiunea ""Extensii de comenzi"" în secțiunea de ajutor a programului\n\
+    CMD.EXE.
 \n\
-    When Extended Features are enabled, the following options are available:\n\
-    /C       Clear the screen before displaying a page.\n\
-    /P       Expand Form-Feed characters.\n\
-    /S       Shrink successive blank lines to a single line.\n\
-    /Tn      Expand tabs to n spaces (default: 8).\n\
+Atunci când opțiunea Caracteristici extinse e activată, sunt disponibile\n\
+următoarele opțiuni:
+    /C       Curăță ecranul înainte de a se afișa o pagină.\n\
+    /P       Extindeți caracterele Form-Feed (de alimentare de formulare).\n\
+    /S       Reduceți liniile goale succesive la o singură linie.\n\
```

Comment:
The same [imperative mood](https://en.wikipedia.org/wiki/Imperative_mood) is used in the above two lines whereas [indicative mood](https://en.wikipedia.org/wiki/Realis_mood) should have been used instead.
Just to clarify through an example, the line below this, the one starting with "extinde", can be both imperative and indicative, whereas the above two lines can be interpreted only in imperative mood.